src/collection-browser.ts

```diff
@@ -1042,6 +1042,7 @@ export class CollectionBrowser
 
     return html` <tile-dispatcher
       .baseNavigationUrl=${this.baseNavigationUrl}
+      .imageBaseUrl=${'https://archive.org'}
```

Comment:
Suggestion: Move this default setting up to `app-root.ts:165`, pass it through `<collection-browser>` to <tile-dispatcher>
```
<collection-browser
    .baseNavigationUrl=${'https://archive.org'}
    .imageBaseUrl=${'https://archive.org'}
```

Reply:
Suggestion rename to `baseImageUrl`